Compile with -fvisibility-inlines-hidden to reduce
the amount of exported symbols that we need to take
care of in the .symbols file.
Index: libqtpas/Qt5Pas.pro
===================================================================
--- a/Qt5Pas.pro
+++ b/Qt5Pas.pro
@@ -21,7 +21,7 @@ TEMPLATE = lib
 VPATH = src
 MOC_DIR = tmp
 OBJECTS_DIR = tmp
-QMAKE_CXXFLAGS += -Wfatal-errors
+QMAKE_CXXFLAGS += -Wfatal-errors -fvisibility-inlines-hidden
 
 # Match Intel x86_64 i686 i586 i386 x86 ...
 is86 = $$find(QMAKE_HOST.arch, ".*86")
